cmd/performance-profile-creator/cmd/root.go

```diff
@@ -51,7 +52,23 @@ var rootCmd = &cobra.Command{
 
 		matchedNodes, err := profilecreator.GetMatchedNodes(nodes, labelSelector)
 		for _, node := range matchedNodes {
-			log.Infof("%s is targetted by %s MCP", node.GetName(), mcpName)
+			nodeName := node.GetName()
+			log.Infof("%s is targetted by %s MCP", nodeName, mcpName)
+			snapShotOptions, err := profilecreator.LoadSnapshot(mustGatherDirPath, node)
```

Comment:
either we rename `LoadSnapshot` (which is not - it is creating the right options to consume the snapshot) or we add a thin wrapper around ghw to get the API object we care about from snapshot, node, parameters. Something like:

```
handle, err := profilecreator.NewKNI(mustGatherDirPath, node)
...

cpuInfo, err := handle.CPU()
...
topologyInfo, err := handle.Topology()
```
I'm fine with either approaches

Reply:
@marcel-apf Do you have any preference? Is it okay to just update the name to `ConsumeSnapshot`  for now?

Reply:
Yes, it is, however I prefer the "handle" approach. I am good with both options though.

Reply:
Done!